doc/init.md

```diff
@@ -20,9 +20,9 @@ The macOS configuration assumes bitcoind will be set up for the current user.
 Configuration
 ---------------------------------
 
-At a bare minimum, bitcoind requires that the rpcpassword setting be set
-when running as a daemon.  If the configuration file does not exist or this
-setting is not set, bitcoind will shut down promptly after startup.
+Running bitcoind as a daemon does not require any manual configuration. You may
+set the rpcpassword setting in the `bitcoin.conf` configuration file to override
+the default behaviour of using a special cookie for authentication.
```

Comment:
Maybe give the (relative) path for cookie file

Reply:
There is a dedicated `doc/files.md` file on this purpose in the repo.